data/json/items/gun/308.json

```
@@ -446,5 +455,48 @@
     "min_cycle_recoil": 2700,
     "pocket_data": [ { "pocket_type": "MAGAZINE_WELL", "item_restriction": [ "ar10mag_20rd", "ar10_makeshiftmag" ] } ],
     "melee_damage": { "bash": 12 }
+  },
+  {
+    "id": "rfb_308",
+    "copy-from": "rifle_semi",
+    "looks_like": "ar15",
+    "type": "GUN",
+    "name": { "str": "7.62x51mm bullpup rifle" },
+    "description": "This is a semi-automatic bullpup rifle with a lightened body due to the use of polymers and aluminum, using standard magazines of the heavy automatic rifle and a 7.62x51mm cartridge.",
+    "variant_type": "gun",
+    "variants": [
+      {
+        "id": "rfb_308",
+        "name": { "str_sp": "Kel-Tec RFB" },
+        "description": "This is a semi-automatic bullpup rifle with a lightened body due to the use of polymers and aluminum, using magazines from the FN FAL and the .308 Winchester cartridge.  The main design features include forward case ejection, the duplication of the magazine release button and safety catch for left-handers, and a heavy barrel serving as the gun's spine.  The most noticeable features of the RFB are reflected in the transcription of its name: rifle, forward-ejection, bullpup.",
```

The second and last sentence seem awkward. You could combine them... I also don't think there's major use of aluminum in the design, it's mostly the polymer that stands out here.
```suggestion
        "description": "This is a semi-automatic bullpup rifle with a lightweight, polymer frame, using magazines from the FN FAL and the .308 Winchester cartridge.  The Rifle Forward-ejection Bullpup (RFB) features forward case ejection, the ambidextrous controls, and structural use of the barrel, uncommon in modern designs.",
```